packages/components/src/local/route/styles.module.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file can be deleted if it is empty. As can `styles.moduile.scss.d.ts` just make sure it's not referenced anywhere.